gen-models/gen-models.capnp

```diff
@@ -196,20 +196,21 @@ struct DatabaseChangeset {
 
 struct ManifestOperation {
   operation @0 :Operation;
-  changesetHash @1 :Text;
```

Comment:
I realized these fields aren't used and were adding some extra work in finding the right files on remotes for testing so i removed them.